docs/changelog.rst

```diff
@@ -2,13 +2,19 @@ CbAPI Changelog
 ===============
 .. top-of-changelog (DO NOT REMOVE THIS COMMENT)
 
+Updates Dec 13, 2021
```

Comment:
Are you planning on pushing a new version as we don't update the changelog with a new entry unless it's a new version?

Reply:
removed from the change log.